packages/node/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+- Do not index data when fullBlock does not exist logs (#350)
```

Comment:
This doesn't make sense to me. It should still be possible to index a block without logs